core/src/main/java/org/elasticsearch/cluster/metadata/IndexMetaData.java

```diff
@@ -256,6 +259,7 @@ private IndexMetaData(String index, long version, State state, Settings settings
             }
         }
         useTypeForRouting = settings.getAsBoolean(SETTING_LEGACY_ROUTING_USE_TYPE, false);
+        indexSlowLoggingSource = settings.getAsBoolean(SETTING_SLOW_LOGGING_SOURCE, true);
```

Comment:
And just so that I put a comment on every line, maybe logSourceInIndexSlowLog? This setting makes me think it has to do with the source of the slow log..... I know I'm picky.